### DISCUSSION TITLE
Ventoy Browser from other bootloaders(e.g. grub4dos/grub2/rEFInd/...) like vdiskchain, Grub2 File Manager

### DISCUSSION THREAD

**Discussion by candrapersada** [ventoy/Ventoy#1567] [Ideas]
is it possible to add Ventoy Browser to other bootloaders(e.g. grub4dos/grub2/rEFInd/...) like vdiskchain

**Comment by ventoy** [comment]
NO.
Ventoy can not work as a virtual disk image.
If you want to use Ventoy, you must install Ventoy to a physical disk (USB, HDD ...)

**Comment by candrapersada** [comment]


**Reply by ventoy** [reply]
NO

**Comment by candrapersada** [comment]
add feature to chainload Ventoy Linux vDisk file from other bootloaders(e.g. grub4dos/grub2/rEFInd/...) like [Ventoy Boot Image Files In Local Disk](https://www.ventoy.net/en/doc_vlnk.html) and [Browse/Boot Files In Local Disk](https://www.ventoy.net/en/doc_browser.html) without install Ventoy to a physical disk
